CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Next (Unreleased)
 
+## v.0.3.1 (October 1st, 2024)
+
+* Attach competition in Kaggle notebook environment ([#161](https://github.com/Kaggle/kagglehub/pull/161))
+* Download competition using `kagglehub.competition_download(...)` ([#158](https://github.com/Kaggle/kagglehub/pull/158))
+* Enable authorization via Colab secrets ([#157](https://github.com/Kaggle/kagglehub/pull/157))
```

Comment:
@dolaameng are okay with your work with colab secret should be included in change log, not sure if it a work in progress feature

Reply:
LGTM! Thank you @calderjo !